worker.go

```diff
@@ -592,6 +628,18 @@ func (r *Result) PrettyPrint() string {
 			}
 		}
 	}
+
+	// Recommended ER, only in ER calc mode
+	if r.RequiredER[0] != 0 {
```

Comment:
not sure why here you are checking if the first element does not equal 0?

shouldn't you be checking if ER mode?

Reply:
Fixed